box.json.dist

```diff
@@ -0,0 +1,36 @@
+{
+    "check-requirements": false,
+    "dump-autoload": false,
+    "stub": "php-cs-fixer",
+    "chmod": "0755",
+    "compactors": [
+        "Herrera\\Box\\Compactor\\Json",
+        "Herrera\\Box\\Compactor\\Php"
+    ],
+    "files": [
+        "dev-tools/ci-integration.sh",
+        "LICENSE"
+    ],
+    "finder": [
+        {
+            "name": [
+                "*.php",
+                "ci-integration.sh"
```

Comment:
shall it be indeed, in `finder` and `files` ?

Reply:
No need indeed :)